src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: $cardWidth;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  font-weight: 400px;
```

Comment:
1 - font-weight is a value without units, it's not working now
2 - it's a default value, so it can be removed